internal/workflow_testsuite.go

```diff
@@ -449,6 +449,13 @@ func (c *MockCallWrapper) Times(i int) *MockCallWrapper {
 	return c
 }
 
+// Zero indicates that the mock should not be called.
+func (c *MockCallWrapper) Zero() *MockCallWrapper {
```

Comment:
```suggestion
// Never indicates that the mock should not be called.
func (c *MockCallWrapper) Never() *MockCallWrapper {
```

I think never might be a slightly more intuitive name here, and is fairly common in mocking libs.

Reply:
Done!!